setup.py

```diff
@@ -1,30 +1,6 @@
-import setuptools
+#!/usr/bin/env python
```

Comment:
Do you care about or enforce python 3.x as a requirement? e.g. on many gentoo and centos systems, `python` is actually `python2.7`. Not sure what the pythonic solution is, but in Linux I'd do something like `#!/usr/bin/env python3`

Reply:
Derp and great suggestion!